openapi.yaml

```diff
@@ -1676,6 +1676,21 @@ components:
           type: number
           example: 1
           description: 'Revision specifying the updates the record has gone through'
+        userRole:
+          type: string
+          example: 'researcher'
+        projectId:
```

Comment:
Project Id is an array of project ids?

Reply:
Thanks for catching that, fixed in next revision :)